index.js

```diff
@@ -76,6 +77,19 @@ module.exports = function (str, opts) {
 		urlObj.pathname = urlObj.pathname.replace(/\/{2,}/g, '/');
 	}
 
+	// remove directory index
+	if (opts.removeDirectoryIndex === true) {
+		opts.removeDirectoryIndex = [/^index/];
+	}
+
+	if (Array.isArray(opts.removeDirectoryIndex) && opts.removeDirectoryIndex.length) {
```

Comment:
That wouldn't be right. The first `if` just sets the default value for the value `true`. It does not proceed to modify the path name. Hence the two aren't mutually exclusive.